internal/resource/statefulset_test.go

```diff
@@ -975,7 +978,7 @@ default_pass = {{ .Data.data.password }}
 									"memory": k8sresource.MustParse("512Ki"),
 								},
 							},
-							Image: instance.Spec.SecretBackend.CredentialUpdaterImage,
+							Image: "rabbitmqoperator/admin-password-updater:0.1.1",
```

Comment:
That's brittle because the unit test will fail when we bump the default version.

Reply:
That is true. I was going to check against non-emtpy but given that we always check against full strings I thought it was more appropriate. But yes, it is more brittle because we have to update the unit tests too.

Reply:
Fixed in https://github.com/rabbitmq/cluster-operator/pull/846/commits/9b9a3d0d61f0ed838b279ac767dcdbbdc236a0b2